packages/server/src/utils/S3.ts

```diff
@@ -1,21 +1,22 @@
 import AWS from 'aws-sdk';
 import { GetObjectOutput } from 'aws-sdk/clients/s3';
-import readline from 'readline';
 import { isDev } from '../lib/env';
-import { logError } from './logging';
 
-if (isDev) {
-    AWS.config.credentials = new AWS.SharedIniFileCredentials({
-        profile: 'membership',
-    });
-}
-const S3 = new AWS.S3();
+const getS3 = () => {
```

Comment:
I think this is definitely the most straightforward solution; it'll be worth keeping an eye on performance just in case we see a dip in response times, but unless it's having a genuine impact I don't think it's worth introducing the complexity and/or amount of required refactoring of sharing around a single client instance.

Reply:
👍 agreed, especially given the amount of caching we do

Reply:
also, if we find it is creating an excessive amount of s3 clients then we should prioritise the work to improve the cache - https://trello.com/c/Hx4gVamE/235-cache-switches-better-in-sdc